packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -70,7 +71,7 @@ export type SuiteAction =
       }
     | {
           type: typeof SUITE.SET_THEME;
-          variant: SuiteThemeVariant;
+          variant: Exclude<SuiteThemeVariant, 'system'>;
```

Comment:
Please exclude 'system' also from the setTheme function param below.

Reply:
changed to `AppState['suite']['settings']['theme']['variant']` to bind action with expected value in reducer